evergreen.yaml

```diff
@@ -48,8 +48,20 @@ functions:
         include_expansions_in_env: ["MONGODB_URL"]
         env:
           DECOMPRESS: ${decompress}
+          MONGODB_URL: ${MONGODB_URL}
+          MONGODB_DECOMPRESS: ${MONGODB_DECOMPRESS|*decompress}
         working_dir: anser
         command: make get-mongodb
+    - command: subprocess.exec
+      type: setup
+      params:
+        command: make get-mongosh
+        include_expansions_in_env: ["MONGOSH_DOWNLOAD"]
```

Comment:
this seems like this could have been left over

Reply:
Fixed